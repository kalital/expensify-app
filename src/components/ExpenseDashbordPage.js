import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './EXpenseListFilter'

const ExpenseDashbordPage = () => (
     <div>
        <ExpenseListFilters />
        <ExpenseList />
     </div>
 );

 export default ExpenseDashbordPage;