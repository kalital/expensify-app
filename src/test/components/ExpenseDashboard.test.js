import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashbordPage from '../../components/ExpenseDashbordPage'

test('should render ExpenseDashbordPage correctly', () => {
 const wrapper = shallow(<ExpenseDashbordPage />);
expect(wrapper).toMatchSnapshot();
}); 