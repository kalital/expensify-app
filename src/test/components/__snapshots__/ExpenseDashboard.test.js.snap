// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDashbordPage correctly 1`] = `
<div>
  <[object Object] />
  <[object Object] />
</div>
`;
